changelog/v1.14.0-beta10/fix-istio-integration-headless-service-crash.yaml

```
@@ -0,0 +1,11 @@
+changelog:
+  - type: FIX
+    description: Use Pod IPs for headless services when the gloo deployment's `ENABLE_ISTIO_INTEGRATION` is `true`.
+    issueLink: https://github.com/solo-io/gloo/issues/7491
+    resolvesIssue: false
+  - type: FIX
+    description: |
+      Fix a bug where the validation webhook would still start even if the `VALIDATION_MUST_START` environment variable was `false`.
```

nit, just for clarity
```suggestion
      Fix a bug where the validation webhook server on the gloo pod would still start even if the `VALIDATION_MUST_START` environment variable was `false`.
```